plugins/modules/postgresql_query.py

```diff
@@ -363,6 +363,14 @@ def set_search_path(cursor, search_path):
     cursor.execute('SET search_path TO %s' % search_path)
 
 
+def insane_query(string):
+    for c in string:
+        if c not in (' ', '\n', '', '\t', ','):
```

Comment:
why the `','`

Reply:
i've no idea:) yep, it's obviously incorrect, would you like to submit a code change here so that i merge the commit?

Reply:
i mean to remove the `','` from the list